### PR TITLE
Add links to GPG and PGP tutorials

### PR DESCRIPTION
**Pull request by shifterovich2**
### Description

#347 

### HTML Preview

http://htmlpreview.github.io/?https://github.com/shifterovich2/privacytools.io/blob/tutorial-buttons/index.html

### CODE REVIEW

**Comment by Shifterovich** [comment]
@hugoncosta

**Comment by hugoncosta** [comment]
Yeah, works. But wouldn't it be better if we used a different colour? That way people would understand that there's something different that they might want to check out other than the website.

**Comment by Shifterovich** [comment]
Good point. Any suggestions?

**Comment by hugoncosta** [comment]
I was looking at them and the colours I was thinking about were already on the other panels. Maybe btn-secondary (gray)?

**Comment by shifterovich2** [comment]
Yeah, I was thinking B/W colors would somewhat match the tutorial website's logo.

File: index.html [hunk 1]
```diff
@@ -1282,9 +1282,9 @@ <h3 class="panel-title">GNU IceCat</h3>
 
 		<h3>Privacy Email Tools</h3>
 		<ul>
-			<li><a href="http://www.gpg4usb.org/">gpg4usb</a> - A very easy to use and small portable editor to encrypt and decrypt any text-message or -file. For Windows and Linux.</li>
-			<li><a href="https://www.mailvelope.com/">Mailvelope</a> - A browser extension that enables the exchange of encrypted emails following the OpenPGP encryption standard.</li>
-			<li><a href="https://www.enigmail.net/">Enigmail</a> - A security extension to Thunderbird and Seamonkey. It enables you to write and receive email messages signed and/or encrypted with the OpenPGP standard.</li>
+			<li><a href="http://www.gpg4usb.org/">gpg4usb</a> - A very easy to use and small portable editor to encrypt and decrypt any text-message or -file. For Windows and Linux. <a href="https://theprivacyguide.org/tutorials/gpg.html">GPG tutorial</a>.</li>
+			<li><a href="https://www.mailvelope.com/">Mailvelope</a> - A browser extension that enables the exchange of encrypted emails following the <a href="https://theprivacyguide.org/tutorials/pgp.html">OpenPGP encryption standard</a>.</li>
```
Comment (shifterovich2) [inline index.html]: Are these links okay as well?

**Comment by shifterovich2** [comment]
See my comment in the review.

**Comment by victorhck** [comment]
the emailselfdefense it's a great resource created by FSF and maintained and translated into many languages by supporters:
https://emailselfdefense.fsf.org/
Maybe it's a good resource to include

File: index.html (hunk 1 above)
Comment (hugoncosta) [reply index.html]: Yeah that looks nice! They're lowkey, but those that are really interested in knowing how to use them will appreciate them.

**Comment by Shifterovich** [comment]
@victorhck Not specifically a GPG tutorial, but definitely worth adding it to the site. Maybe the resources section at the bottom of the site?

**Comment by Shifterovich** [comment]
@hugoncosta Seems `.btn-secondary` is Bootstrap 4. We could use `.btn-default` or update to Bootstrap 4.

**Comment by hugoncosta** [comment]
Sorry, forgot to answer. I didn't check, we can stick to the default one, my idea was to have a different colour, and that one does the job

**Comment by Shifterovich** [comment]
Since no other collaborator seems to have time, I'll consider your approval a collaborator approval and merge.

**Pull request merged**